application/helpers/update/updatedb_helper.php

```diff
@@ -1339,6 +1339,7 @@ function db_upgrade_all($iOldDBVersion, $bSilent = false)
         if ($iOldDBVersion < 256) {
             $oTransaction = $oDB->beginTransaction();
             upgradeTokenTables256();
+            $oDB->createCommand()->update('{{participants}}', array('email' => ''), 'email IS NULL');
```

Comment:
Think is not related to this issue …


I do same … staring from a fork to do another fork : https://github.com/LimeSurvey/LimeSurvey/pull/1370#issuecomment-578443506 :)

Reply:
If this is not related to the issue, please remove from this PR and add it to another one.